lib/cmd/fh3/connections/list.js

```diff
@@ -0,0 +1,29 @@
+/* globals i18n */
+var common = require("../../../common");
+
+module.exports = {
+  'desc' : i18n._('List connections'),
+  'examples' :
+  [{
+    cmd : 'fhc connections list --project=<project>',
+    desc : i18n._('List all connections of the <project>')
+  }],
+  'demand' : ['project'],
+  'alias' : {
+    'project' : 'p',
+    0 : 'project',
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this trailing comma. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATrailingComma)